clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/HeartbeatMonitoringServiceStats.java

```diff
@@ -0,0 +1,22 @@
+package com.linkedin.davinci.stats;
+
+import com.linkedin.venice.stats.AbstractVeniceStats;
+import io.tehuti.metrics.MetricsRepository;
+import io.tehuti.metrics.Sensor;
+import io.tehuti.metrics.stats.Count;
+
+
+public class HeartbeatMonitoringServiceStats extends AbstractVeniceStats {
+  private static final String HB_SUFFIX = "-heartbeat-monitor-service";
+  private final Sensor heartbeatExceptionCountSensor;
+
+  public HeartbeatMonitoringServiceStats(MetricsRepository metricsRepository, String clusterName) {
+    super(metricsRepository, clusterName + HB_SUFFIX);
+    heartbeatExceptionCountSensor = registerSensorIfAbsent("heartbeat-monitor-service-exception-count", new Count());
```

Comment:
Can you have a heart-beat metric as well?
If the thread is not running, the heart-beat metric will stop emitting any data, which can be a signal we can alert against.
One example:
https://github.com/linkedin/venice/blob/main/clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/ParticipantStoreConsumptionTask.java#L64

Reply:
Sure, a heartbeat for heartbeat services :)